analyzers/src/SonarAnalyzer.CSharp/Facade/CSharpSyntaxFacade.cs

```diff
@@ -81,5 +82,17 @@ node switch
                 LiteralExpressionSyntax literalExpression => literalExpression.Token.ValueText,
                 _ => string.Empty
             };
+
+        public override SyntaxNode RemoveParentheses(SyntaxNode expression)
+        {
+            var current = expression;
+            while (current?.IsAnyKind(SyntaxKind.ParenthesizedExpression, SyntaxKindEx.ParenthesizedPattern) ?? false)
```

Comment:
The old implementation `? .. ?? false` is somehow overengineered. Can you please update it to `current != null && current.IsAnyKind()) to simplify it?

Reply:
Done both.